src/libraries/System.Net.Sockets/tests/FunctionalTests/UnixDomainSocketTest.cs

```diff
@@ -90,33 +90,31 @@ public async Task Socket_ConnectAsyncUnixDomainSocketEndPoint_NotServer()
         {
             string path = GetRandomNonExistingFilePath();
             var endPoint = new UnixDomainSocketEndPoint(path);
-            try
-            {
-                SocketAsyncEventArgs args = new SocketAsyncEventArgs();
-                args.RemoteEndPoint = endPoint;
-                args.Completed += (s, e) => ((TaskCompletionSource)e.UserToken).SetResult();
+            SocketAsyncEventArgs args = new SocketAsyncEventArgs();
+            args.RemoteEndPoint = endPoint;
+            args.Completed += (s, e) => ((TaskCompletionSource)e.UserToken).SetResult();
 
-                var complete = new TaskCompletionSource();
-                args.UserToken = complete;
+            var complete = new TaskCompletionSource();
+            args.UserToken = complete;
 
-                using (Socket sock = new Socket(AddressFamily.Unix, SocketType.Stream, ProtocolType.Unspecified))
+            using (Socket sock = new Socket(AddressFamily.Unix, SocketType.Stream, ProtocolType.Unspecified))
+            {
+                bool willRaiseEvent = sock.ConnectAsync(args);
+                if (willRaiseEvent)
                 {
-                    bool willRaiseEvent = sock.ConnectAsync(args);
-                    if (willRaiseEvent)
-                    {
-                        await complete.Task;
-                    }
+                    await complete.Task;
 
                     Assert.Equal(
                         OperatingSystem.IsWindows() ? SocketError.ConnectionRefused : SocketError.AddressNotAvailable,
                         args.SocketError);
                 }
+
+                Assert.Equal(
+                    RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ? SocketError.ConnectionRefused : SocketError.AddressNotAvailable,
+                    args.SocketError);
             }
-            finally
-            {
-                try { File.Delete(path); }
-                catch { }
-            }
+
+            Assert.False(File.Exists(path));
```

Comment:
This assert fails on Windows. That means either .NET, or Windows kernel creates a file when Connect is performed. This doesn't happen on Linux. I'm going to remove the assert because it is not needed to validate the PR.